src/__tests__/element-queries.js

```diff
@@ -332,6 +347,61 @@ test('label with no form control', () => {
 `)
 })
 
+test('label with no form control and fuzzy matcher', () => {
+  const {getByLabelText, queryByLabelText} = render(
+    `<label>All alone label</label>`,
+  )
+  expect(queryByLabelText('alone', {exact: false})).toBeNull()
+  expect(() => getByLabelText('alone', {exact: false}))
+    .toThrowErrorMatchingInlineSnapshot(`
+"Found a label with the text of: alone, however no form control was found associated to that label. Make sure you're using the "for" attribute or "aria-labelledby" attribute correctly.
+
+<div>
+  <label>
+    All alone label
+  </label>
+</div>"
+`)
+})
+
+test('label with children with no form control', () => {
+  const {getByLabelText, queryByLabelText} = render(`
+  <label>
+    All alone but with children
+    <textarea>Hello</textarea>
+    <select><option value="0">zero</option></select>
+  </label>`)
```

Comment:
Maybe I am not understanding, you are saying that the right test to be done here (at line 374) was that the labeled element is `textarea` instead of that no `input` element labeled was found?

`getByLabelText(/alone/)` would have found the `textarea` element, I added the selector to make it return null.

Please tell me if I am missing something.